server/index.js

```diff
@@ -1,15 +1,18 @@
 const express = require('express');
 const path = require('path');
 require('dotenv').config();
+const db = require('../database/index');
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(express.json());
 app.use(express.urlencoded());
 
-app.get('/api/reviews', (req, res) => {
-  res.send('testing GET request on /api/reviews endpoint');
+app.get('/api/reviews/:id', (req, res) => {
+  db.getReviews(req.params.id)
+    .then((rows) => res.json(rows))
```

Comment:
Looks good. You may also want to consider adding a status code for the success and fail responses.